src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -43,6 +43,7 @@ class ImageBaseViewer extends BaseViewer {
             return;
         }
 
+        ImageBaseViewer.setOriginalImageSize(this.imageEl);
```

Comment:
We don't have a policy one way or another, but I'd lean towards making this a normal private function in the base viewer until otherwise needed since that's how we've organized other similar helper functions. The moment this is required in a different viewer outside of the ImageBaseViewer inheritance chain, we can easily update this to a static method.